Rent.Service/Rent.Service.API/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Rent.Service.API.Dtos;
+using Rent.Service.Application.Common.Exceptions;
+using System.Net;
+
+namespace Rent.Service.API.Middleware;
+
+public class ErrorHandlingMiddleware(RequestDelegate next)
```

Comment:
Why this class only handle one error and response with only bad request. Try create somthing more flexible for more error and responses in future